client/ayon_houdini/plugins/load/load_filepath.py

```diff
@@ -25,6 +25,10 @@ class FilePathLoader(plugin.HoudiniLoader):
     product_types = {"*"}
     representations = {"*"}
 
+    def _add_more_node_params(self, attr_folder, node):
```

Comment:
we can move this function outside of the class as it is just used in `NodePresetLoader`? We dont need to pass the function here if we do that.
```suggestion
    def _add_more_node_params(self, attr_folder, node):
```

Reply:
It's actually used inside `FilePathLoader` in the middle of  `FilePathLoader.load` method. 
Then I override it in `NodePresetLoader` to add the extra custom parameters to the `Attributes` folder (`attr_folder`). 

I chose this way to avoid copying/pasting the code of `FilePathLoader`

---

I thought about using a different approach where I do something like this in the subclass `NodePresetLoader` .

```python
class NodePresetLoader(FilePathLoader):
    label = "Load Node Preset"
    order = 9
    icon = "link"
    color = "white"
    product_types = {"node_preset"}
    representations = {"json"}

    def load(self, context, name=None, namespace=None, data=None):
      container = super(NodePresetLoader, self).load(context, name=name, namespace=namespace, data=data)
      node = container["node"]
      parm_group = node.parmTemplateGroup()
      attr_folder = parm_group.find("attributes_folder") 
      ## But this won't work because Houdini overrides the parameter name.
```
![image](https://github.com/user-attachments/assets/8a326457-716e-46ef-8ce0-745d74488997)

Reply:
In regards to your second approach, I think your first approach is better.